buildSetup.sc

```diff
@@ -159,19 +162,51 @@ trait BaseMunitTests extends ScalafmtModule with TestModule.Munit {
     )
 }
 
-case class ScalaVersion(maj: Int, min: Int, patch: Int)
-object ScalaVersion {
-  def apply(scalaVersion: String): ScalaVersion = scalaVersion match {
+trait MimaModule extends Mima {
+
+  def mimaPreviousVersions = getVersionsFromTags
+
+  override def mimaCheckDirection = T { CheckDirection.Backward }
+
+  def baseMimaVersion: Version
```

Comment:
Perhaps this should be `def baseMimaVersion: T[Version]` to give more flexibility with mill and e.g. being able to see it from the mill CLI